src/app/campaign/store/reducers/flight.reducer.ts

```diff
@@ -59,7 +59,7 @@ const _reducer = createReducer(
         set_inventory_uri: null
       },
       changed: false,
```

Comment:
This right here was the new flight bug. Ryan pointed a few of these out in PRs earlier in the project, think I wouldn've gotten that sorted by now.